dev_scripts/find_images_to_rebuild.py

```diff
@@ -40,15 +46,30 @@
 
 errata = Errata()
 kwargs = {}
-if container_images:
+
+if args.container_img:
     EventClass = ManualRebuildWithAdvisoryEvent
-    kwargs['container_images'] = container_images
+    kwargs['container_images'] = args.container_img
 else:
     EventClass = ErrataAdvisoryStateChangedEvent
+
+event_id = None
+if args.cassette:
+    cassette_name = os.path.splitext(os.path.basename(args.cassette))[0]
+    try:
+        event_id = int(cassette_name)
+    except ValueError:
+        print("The input cassette must be named in the format of <event ID>.yaml")
```

Comment:
This was my mistake. I thought 687072cfba2d2474bbf72e4a4359c8c3df794ddf used `.yaml`, but it's actually `.yml`.
```suggestion
        print("The input cassette must be named in the format of <event ID>.yml")
```